includes/modules/ezpages_bar_footer.php

```diff
@@ -17,14 +17,20 @@
 
 // test if bar should display:
 if (EZPAGES_STATUS_FOOTER == '1' || (EZPAGES_STATUS_FOOTER == '2' && (strstr(EXCLUDE_ADMIN_IP_FOR_MAINTENANCE, $_SERVER['REMOTE_ADDR'])))) {
-  $pages_query = $db->Execute("SELECT e.pages_id, e.page_open_new_window, e.page_is_ssl, e.alt_url, e.alt_url_external, e.toc_chapter, ec.pages_title
+
+  if (defined('TABLE_EZPAGES_CONTENT')) { 
+    $pages_query = $db->Execute("SELECT e.pages_id, e.page_open_new_window, e.page_is_ssl, e.alt_url, e.alt_url_external, e.toc_chapter, ec.pages_title
                               FROM  " . TABLE_EZPAGES . " e,
                                     " . TABLE_EZPAGES_CONTENT . " ec
                               WHERE e.pages_id = ec.pages_id 
                               AND ec.languages_id = " . (int)$_SESSION['languages_id'] . "
                               AND e.status_footer = 1
                               AND e.footer_sort_order > 0
                               ORDER BY e.footer_sort_order, ec.pages_title");
+  } else {
+    // Failing query 
+    $pages_query = $db->Execute("SELECT * FROM " . TABLE_EZPAGES . " WHERE pages_id < 0"); 
```

Comment:
The test for `pages_id < 0` is going to just give empty results ... which in itself is an okay way to "obtain a database response, and pass the empty response on to the rest of the code.

But I'd rather see us do something like an "early return" if things are "in an old unupgraded-and-therefore-unusable-state", since then that part of the code can be easily identified and removed later when someone comes along wanting to remove deprecated code/comments.